frontends/ast/simplify.cc

```diff
@@ -822,11 +863,68 @@ bool AstNode::simplify(bool const_fold, bool at_zero, bool in_lvalue, int stage,
 			range_swapped = templ->range_swapped;
 			range_left = templ->range_left;
 			range_right = templ->range_right;
+			attributes["\\wiretype"] = mkconst_str(resolved_type->str);
```

Comment:
New attributes should be added to the attributes section of the README file.

Reply:
added some documentation